### ISSUE TITLE
VPAT -- Layout Tables (Customize Website Theme)

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#2936]
`<table>` is typically used to present information in tabular form, but is also commonly used for page layout.  For accessibility, the two can be distinguished by the use, or lack of, table headers.

Data tables should have `<th>`, presumably also enclosed with `<thead>` ... `</thead>`.  Also a `<caption>` should be provided before `<thead>`.  Or possibly use `<caption />` if there is no caption to display.

Layout tables do not have explicit headings or caption in HTML.  Therefore, they are identified by the absence of `<thead>`, `<th>`, and `<caption>`.

The Customize Website Theme view of Course Configuration appears to have a _layout_ table (for uploading HTML files) but table headers (`<th>`) is being used for visual styling for boldface fonts.  As this table seems to have more to do with layout than data, boldface visual styling should be enforced with CSS, not with HTML table headers.

Per WAVE regarding **Data** Tables
> **What It Means**
> A data table is present.
> **Why It Matters**
> Data tables present tabular data. Data tables should contain table header cells that identify the content of their respective row and/or columns. Tables with proper table headers provide additional information and navigation for screen reader users.
> **How to Fix It**
> Ensure that the table contains tabular data and that it is not used merely for page layout. Ensure that all column and row headers are `<th>` elements and ensure the data cells are associated with their proper header cells (typically by assigning scope to the table headers). Where appropriate, associate a descriptive caption (`<caption>` element) to the table.
> **The Algorithm... in English**
> A `<table>` element is present that contains at least one table header cell (`<th>`).
> **Standards and Guidelines**
> [Section 508 (g)](http://webaim.org/standards/508/checklist#standardg)
> [Section 508 (h)](http://webaim.org/standards/508/checklist#standardh)
> [1.3.1 Info and Relationships (Level A)](http://webaim.org/standards/wcag/checklist#sc1.3.1)

Per WAVE regarding **Layout** Tables
> **What It Means**
> A layout table is present.
> **Why It Matters**
> While tables are primarily intended for the presentation of tabular information or data, they are often used to control page layout and formatting. Layout tables can introduce reading and navigation order issues and must not contain header (`<th>`) cells.
> **How to Fix It**
> Ensure that the table is indeed a layout table and that it does not contain tabular data. If it is a data table, provide appropriate header (`<th>`) cells. Verify that the reading and navigation order of table content (based on underlying source code order) is logical.
> **The Algorithm... in English**
> A `<table>` element is present that does not contain any header (`<th>`) cells.
> **Standards and Guidelines**
> None
